en/lessons/libraries/guardian.md

```diff
@@ -230,26 +267,4 @@ def delete(conn, params) do
 end
 ```
 
-When using API login, it's slightly different because there's no session and you need to provide the raw token back to the client.
-For API login you'll likely use the `Authorization` header to provide the token to your application. This method is useful when you do not intend on using a session.
-
-```elixir
-def create(conn, params) do
-  case find_the_user_and_verify_them_from_params(params) do
-    {:ok, user} ->
-      {:ok, jwt, _claims} = Guardian.encode_and_sign(user, :access)
-      conn
-      |> respond_somehow({token: jwt})
-    {:error, reason} ->
-      # handle not verifying the user's credentials
-  end
-end
-
-def delete(conn, params) do
-  jwt = Guardian.Plug.current_token(conn)
-  Guardian.revoke!(jwt)
-  respond_somehow(conn)
-end
-```
-
-The browser session login calls `encode_and_sign` under the hood so you can use them the same way.
+The sign_in function is ok to use with or without sessions. If there is a session it will store the token in the session but if not it will just create the token and set it on the connection so you can then provide it to your client.
```

Comment:
AFAIK we usually wrap functions with backticks and specify their arities. So I guess [`sign_in`](https://github.com/ueberauth/guardian/blob/master/lib/guardian/plug.ex#L79) in this sentence need a little attention.